docs/source/examples/index.rst

```diff
@@ -19,7 +19,7 @@ A collection of examples demonstrating the use cases of SkyPilot.
    :maxdepth: 2
 
    Quickstart: PyTorch <../getting-started/tutorial>
-   API Server Deployment <skypilot-deployment/index>
+   SkyPilot Deployment <skypilot-deployment/index>
```

Comment:
I intentionally changed it to `API Server Deployment`.

If we use `SkyPilot Deployment`, it confuses new user - "Are all these steps necessary for me to use SkyPilot on GKE"? These steps are relevant only for API server deployment.